background/storage.js

```diff
@@ -321,6 +377,15 @@ function deleteStyle({id, notify = true}) {
   });
 }
 
+function findDupUsercss(style) {
+  // FIXME: use composed index for better performance?
```

Comment:
it shouldn't take more than a few milliseconds so if this code runs only on install/update, there's no need for this fixme.